From: Chad Monroe <chad@monroe.io>
Date: Wed, 18 Mar 2026 12:12:21 -0700
Subject: [PATCH] wifi: mt76: mt7915: remove HOST_BROADCAST_PS_BUFFERING

HOST_BROADCAST_PS_BUFFERING tells mac80211 to buffer broadcast/multicast
frames in host memory and expects the driver to drain the buffer via
ieee80211_get_buffered_bc() which mt7915 doesn't support.

The result is that BC/MC frames are trapped in mac80211's ps->bc_buf
whenever a station is in power save mode then silently dropped.

Signed-off-by: Chad Monroe <chad@monroe.io>
---
 mt7915/init.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mt7915/init.c
+++ b/mt7915/init.c
@@ -426,7 +426,6 @@ mt7915_init_wiphy(struct mt7915_phy *phy
 	ieee80211_hw_set(hw, SUPPORTS_MULTI_BSSID);
 	ieee80211_hw_set(hw, WANT_MONITOR_VIF);
 	ieee80211_hw_set(hw, SUPPORTS_TX_FRAG);
-	ieee80211_hw_set(hw, HOST_BROADCAST_PS_BUFFERING);
 
 	hw->max_tx_fragments = 4;
 
